ci/gpu/build.sh

```diff
@@ -53,7 +53,8 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "ucx-py=${MINOR_VERSION}" \
       "xgboost==1.0.2dev.rapidsai0.13" \
       "rapids-build-env=$MINOR_VERSION.*" \
-      "rapids-notebook-env=$MINOR_VERSION.*"
+      "rapids-notebook-env=$MINOR_VERSION.*" \
+      "treelite=0.92"
```

Comment:
treelite version should be updated in the integration repo.

https://github.com/rapidsai/integration/tree/branch-0.15/conda/recipes

Docs for step by step: https://docs.rapids.ai/maintainers/depmgmt/

Reply:
So `treelite` is being newly added as a Conda package (was integrated into the build process here before) and is tied to `xgboost` a bit, which seems to be listed above as well. Given that should we keep them both here or remove both and add them to the metapackage?

Reply:
- Keep in the meta.yaml
- Remove from gpu/build.sh (having it here for testing is okay)
- Submit PR to integration repo to add it to rapids-build-env
- Once that is merged, it should be picked up in GPU builds again

Reply:
Submitted as PR ( https://github.com/rapidsai/integration/pull/60 ).